js/scripts.js

```diff
@@ -1,63 +1,84 @@
+      projectLink: 'https://github.com/SoniaKarungi/Portfolio',
+    },
+    {
+      name: 'Profesional Art Printing Data',
+      descrtiption:
+        "A daily selection of privately personalized reads; no accounts or sign - ups required.has been the industry 's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+      featuredImg: './img/modal-img.png',
+      technologies: ['html', 'bootstrap', 'ruby'],
+      demoLink: 'https://soniakarungi.github.io/Portfolio/',
+      projectLink: 'https://github.com/SoniaKarungi/Portfolio',
+    },
+    {
+      name: 'Profesional Art Printing Data',
+      descrtiption:
+        "A daily selection of privately personalized reads; no accounts or sign - ups required.has been the industry 's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+      featuredImg: './img/modal-img.png',
+      technologies: ['html', 'bootstrap', 'ruby'],
+      demoLink: 'https://soniakarungi.github.io/Portfolio/',
+      projectLink: 'https://github.com/SoniaKarungi/Portfolio',
+    },
+    {
+      name: 'Profesional Art Printing Data',
+      descrtiption:
+        "A daily selection of privately personalized reads; no accounts or sign - ups required.has been the industry 's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+      featuredImg: './img/modal-img.png',
+      technologies: ['html', 'bootstrap', 'ruby'],
+      demoLink: 'https://soniakarungi.github.io/Portfolio/',
+      projectLink: 'https://github.com/SoniaKarungi/Portfolio',
+    },
+    {
+      name: 'Profesional Art Printing Data',
+      descrtiption:
+        "A daily selection of privately personalized reads; no accounts or sign - ups required.has been the industry 's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+      featuredImg: './img/modal-img.png',
+      technologies: ['html', 'bootstrap', 'ruby'],
+      demoLink: 'https://soniakarungi.github.io/Portfolio/',
+      projectLink: 'https://github.com/SoniaKarungi/Portfolio',
+    },
+    {
+      name: 'Profesional Art Printing Data',
+      descrtiption:
+        "A daily selection of privately personalized reads; no accounts or sign - ups required.has been the industry 's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+      featuredImg: './img/modal-img.png',
+      technologies: ['html', 'bootstrap', 'ruby'],
+      demoLink: 'https://soniakarungi.github.io/Portfolio/',
+      projectLink: 'https://github.com/SoniaKarungi/Portfolio',
+    },
   ];
 
+  // Contact form auto-fill from local storage
+  const email = document.getElementById('email');
+  const fullName = document.getElementById('name');
+  const message = document.getElementById('message');
+
+  const localMsg = JSON.parse(localStorage.getItem('messageObject'));
```

Comment:
Nice! You used `JSON.parse`  method so that the saved string is turned back into an object 🔥